docs/eve-images.md

```diff
@@ -131,3 +131,9 @@ When done, you have a live image file to be used, normally in `dist/<arch>/<file
 To launch eden using the provided image:
 
 1. `eden start --image-file=path/to/your/live-image`
+
+### Overwrite config of EVE
+
+You can add files into config partition of EVE (along with the files that are generated by EdenEden) by copying them into `conf` directory.
+You can select another directory you want with `--eve-config-dir` flag of `eden setup` command. To read more about config files please see
```

Comment:
in fact -- you already have a perfect name ;-)

Reply:
And now I use it)